messages/endpoint-must-use-cdn.md

```diff
@@ -0,0 +1,29 @@
+# `endpoint` must use CDN
+
+`@prismicio/client` uses either a repository name or a repository endpoint to query content from Prismic.
```

Comment:
Could we change "repository endpoint" to "Prismic Rest API V2 repository endpoint", just so it's abundantly clear what is accepted?

> `@prismicio/client` uses either a Prismic repository name or a Prismic Rest API V2 repository endpoint to query content from Prismic.